datasets/huggingface.py

```diff
@@ -24,13 +24,19 @@ class DownloadAndPrepareHuggingFaceDatasetJob(Job):
     https://github.com/huggingface/datasets/issues/4179
     """
 
+    __sis_hash_exclude__ = {"split": None, "token": None}
+
+    from datasets import Split
```

Comment:
You should not import sth in the global scope but inside the job only (i.e. only inside `run`).

Reply:
but then how to import the class type `Split`?

Reply:
Ah for type hinting, you would do sth like:
```python
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from datasets import Split
```
But this is not in the class scope but in global scope.

Reply:
But see my other comment below: Do you really want that? I think `split` would be specified as string usually, or not?

Reply:
Ok done.

Reply:
Sorry I pushed before seeing your message. I think we would use a string usually. Thus, I will remove this import then.